src/assets/scripts/client/game/GameOptions.js

```diff
@@ -54,6 +56,8 @@ export default class GameOptions {
     }
 
     /**
+     * Gets the value of a given game option
+     *
      * @for GameOptions
      * @method get
```

Comment:
this makes me cringe. `get` is a reserved word within es2015+.  I know this is legacy code, but we should really update this method name.